server/src/main/java/com/jungleapp/cs414/server/Move.java

```diff
@@ -0,0 +1,9 @@
+package com.jungleapp.cs414.server;
+
+public class Move {
+
+    String row;
```

Comment:
If we want the row and column format to be String, you need to change this to only two variables, **String currentPosition = "52";** and **String toPosition = "53";** for example. This is how our back-end API works. Also as a side note, John and I are currently working on a board implementation to use only int values for row and column, as this really simplifies our code in back end. The reason why we used String in chess was because of the column description being a-h instead of 0-7. Here, we don't have to bother with strings and may leave everything as integer values.